dev/NavigationView/NavigationView_ApiTests/NavigationViewTests.cs

```diff
@@ -1091,5 +1091,42 @@ public void VerifyHierarchicalNavigationTopModeMenuItemsSourceDoesNotCrash()
                 Content.UpdateLayout();
             });
         }
+
+        [TestMethod]
+        public void VerifyNavigationViewItemToolTip()
+        {
+            RunOnUIThread.Execute(() =>
+            {
+                var navView = new NavigationView();
+
+                // Item with null content
+                var menuItem1 = new NavigationViewItem();
+
+                // Item with empty string as content
+                var menuItem2 = new NavigationViewItem();
+                menuItem2.Content = "";
+
+                // Item with null content and custom tooltip
+                var menuItem3 = new NavigationViewItem();
+                ToolTipService.SetToolTip(menuItem3, "Custom tooltip");
+
+                // Item with non-empty string content
+                var menuItem4 = new NavigationViewItem();
+                menuItem4.Content = "Item 4";
+
+                navView.MenuItems.Add(menuItem1);
+                navView.MenuItems.Add(menuItem2);
+                navView.MenuItems.Add(menuItem3);
+                navView.MenuItems.Add(menuItem4);
+
+                Content = navView;
+                Content.UpdateLayout();
+
+                Verify.AreEqual(null, ToolTipService.GetToolTip(menuItem1), "Item 1's tooltip should have been [null].");
+                Verify.AreEqual(null, ToolTipService.GetToolTip(menuItem2), "Item 2's tooltip should have been [null].");
+                Verify.AreEqual("Custom tooltip", ToolTipService.GetToolTip(menuItem3), "Item 3's tooltip should have been \"Custom tooltip\".");
+                Verify.AreEqual("Item 4", ToolTipService.GetToolTip(menuItem4), "Item 4's tooltip should have been \"Item 4\".");
```

Comment:
@Felix-Dev Do you know why the pane is closed here despite not being explicitly closed? When I am running the test locally, this is how the NavigationView renders when adding a IdleSynchronizer.Wait after layout:

![Image of expanded NavigationView](https://user-images.githubusercontent.com/16122379/96845011-81a5d480-1450-11eb-883b-1deef8dd59f8.png)

Also note that the NavigationView has a width of 1024 and height of 644 which I though were the CI dimensions.

Reply:
I am not entirely sure what I am looking at here since the pane appears to be open as expected in the image above? So without the additional Wait() call to render it to the screen, the pane should also be open (since we are in Auto pane display mode and based on the image we launch in a wide enough window).

Perhaps I just misunderstood you here though.

Reply:
Your previous test implementation relied on the fact that the pane should be closed, otherwise the tooltip shouldn't have been set. However in my test runs the pane was open and thus, the tooltip wasn't set for that item.

Reply:
Ah, I see (hence the timing issue you pointed out in another issue since it was working for me). As noted by you, this has been resolved with the updated API tests since I now explicitly close the pane before verifiying the tooltips.